lib/PuppeteerSharp.Tests.DumpIO/Program.cs

```diff
@@ -17,7 +17,7 @@ public static async Task Main(string[] args)
             using (var browser = await Puppeteer.LaunchAsync(options))
             using (var page = await browser.NewPageAsync())
             {
-                await page.EvaluateFunctionAsync("_dumpioTextToLog => console.log(_dumpioTextToLog)", args[0]);
+                await page.EvaluateFunctionAsync<object>("_dumpioTextToLog => console.log(_dumpioTextToLog)", args[0]);
```

Comment:
We need to support "void" functions. That function won't return a value. I don't think we have to force the user to pick a Type he won't need.

This comment applies to many changes here. If we don't need a result, we don't need a generic.

Reply:
Agreed